internal/shared/common.go

```diff
@@ -0,0 +1,12 @@
+package shared
+
+const (
+	// AnnotationKeyGitSyncInstance Resource metadata annotations (keys and values) used for tracking
+	AnnotationKeyGitSyncInstance = "numaplane.numaproj.io/tracking-id"
+	// SSAManager is the default numaplane manager name used by server-side apply syncs
+	SSAManager = "numaplane-controller"
+	// EnvLogFormat log format that is defined by `--logformat` option
+	EnvLogFormat = "ARGOCD_LOG_FORMAT"
+	// EnvLogLevel log level that is defined by `--loglevel` option
+	EnvLogLevel = "ARGOCD_LOG_LEVEL"
```

Comment:
remove any instances of "ArgoCD"

Reply:
By the way, we currently have a "NUMAPLANE_DEBUG" environment variable which gets set to true or false (if false, then "Info" level assumed). Should we have one environment variable for everything? I haven't looked closely at this logrus logging. So, are we going to have logs of two different formats in our log files or no?

Reply:
Yeah, unfortunately gitops engine requires logrus. We can probably have an issue to move to logrus for our normal logging as well.

Reply:
Sure. Or at least have a single environment variable for both? I'm fine with opening a separate issue for that.

Reply:
It seems they are two different use cases. So will leave it there, and create an issue for it.

Reply:
https://github.com/numaproj-labs/numaplane/issues/127